libstuff/libstuff.cpp

```diff
@@ -2564,7 +2572,7 @@ void SQueryLogClose() {
 
 // --------------------------------------------------------------------------
 // Executes a SQLite query
-int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int64_t warnThreshold, bool skipInfoWarn) {
```

Comment:
NAB question: why did we remove the 'e' param here? Is it really not important for debugging?